deps.edn

```diff
@@ -1,7 +1,7 @@
 {:paths ["src/clj" "src/cljs" "src/cljc" "src/js" "src/gen" "test"]
 
  :deps
- {org.clojure/clojure                   #:mvn{:version "1.10.3"}
+ {org.clojure/clojure                   #:mvn{:version "1.11.0"}
   org.clojure/clojurescript             #:mvn{:version "1.10.879"}
```

Comment:
NB: clojurescript is not updated to 11 yet, that requires changing the version here and also a shadow-cljs version that supports it (iirc shadow-cljs overwrites it).